src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -393,6 +393,18 @@ type ParallelReferenceResolution =
     | On
     | Off
 
+[<RequireQualifiedAccess>]
+type TypeCheckingMode =
```

Comment:
add `///` comment on all type definitions

Reply:
Documentation exists in the .fsi.